src/V1/TemplateTree/Map/Builder.php

```diff
@@ -16,10 +17,32 @@ class Builder implements BuilderInterface
     public function build(): MapInterface
```

Comment:
method: "I throw an exception if there are multiple Template Trees specified and they are not named. If there is a single unnamed Template Tree directory path I give it the name "default".  I throw an exception if there are two or more Template Tree directory paths that have the same name specified. I return a Map of Template Trees that are keyed by their specified name (or the name "default" which I assign) each with a value that is the associated Template Tree directory path."